packages/services-store/src/types/translateAPI.ts

```diff
@@ -78,6 +78,7 @@ export interface MastheadLink {
   url?: string;
   hasMenu?: boolean;
   hasMegapanel?: boolean;
+  megamenuLayout?: string;
```

Comment:
Good catch! Yes.

I remember leaving this as simply `string` because I wanted to ask someone about the wisdom of trying to pull the newly added `MEGAMENU_LAYOUT_SCHEME` type from the `web-components` package. Or if there's a cleaner way to define the valid options once and reference them everywhere.